contrib/debian/control

```diff
@@ -182,10 +183,21 @@ Description: The perf metrics collector for the Netdata Agent
 
 Package: netdata-plugin-debugfs
 Architecture: any
-Depends: ${shlibs:Debends},
+Depends: ${shlibs:Depends},
          netdata (= ${source:Version})
 Pre-Depends: libcap2-bin, adduser
 Conflicts: netdata (<< ${source:Version})
 Description: The debugfs metrics collector for the Netdata Agent
  This plugin allows the Netdata Agent to collect Linux kernel metrics
  exposed through debugfs.
+
+Package: netdata-plugin-systemd-journal
+Architecture: any
+Depends: ${shlibs:Depends},
+         netdata (= ${source:Version})
+Pre-Depends: libcap2-bin, adduser
+Build-Depends: libsystemd-dev
```

Comment:
As this only adds a build dependency, there is no real dependency on `libsystemd0`. The result is that the package will be installed but the required shared libraries may not be available.

I assume this is to codify the desire to only have this work if systemd is actually installed, but I want to make sure we intentionally leave out that dependency.

Reply:
No, the `${shlibs:Depends}` part above adds in dependencies automatically handles this. When that’s present, all the executables in the package get scanned to determine what libraries they linked against, and then dpkg generates a correct list of dependencies based on that and replaces this substitution with that list.

The build dependency part is just there to be sure that it’s clear that _only this part of Netdata_ depends on systemd, not the whole thing.